### PR TITLE
Migrate from react-bootstrap to material

### PR DESCRIPTION
**Pull request by aldovandus**
Pull-Request for `paretOS`

## Description
- Replace react bootstrap with material ui

## Relates to
- #296 EditProfile.tsx react-bootstrap -> react-hook-form migration
- 

## Reviewers
- @person1 (merge duty)
- @person2
- ...

## Steps to reproduce (if describing bug fix)
- Start project
- Make request to `/endpoint` with `{ "body": "test" }`
- Expect some response

## Screenshots / other info
image

### CODE REVIEW

File: src/profile/EditProfile.tsx [hunk 1]
```diff
@@ -56,7 +67,7 @@ const EditProfile = (props: any) => {
     // what we did above, was the get the user id from the navigation bar
     const response = await RestAPI.get("pareto", `/users/${userId}`, {});
     // here we are populating our initial state. In the future, we will likely just pass stuff in via props, instead of running a fresh network request. That was a legacy decision, don't worry about it @antonio-b
-    console.log('u', response);
+    console.log("u", response);
```
Comment (jeremydthomas) [inline src/profile/EditProfile.tsx:70]: Delete this console.log, easy to forget, believe me I know 😄

File: src/profile/EditProfile.tsx [hunk 2]
```diff
@@ -66,10 +77,12 @@ const EditProfile = (props: any) => {
   // This function below handles the changes in state, based on the forms. All of the information stored in the forms, is stored in state. Each form has an `id`, which is accessed by the event.target.id.
   // The actual updated value, is represented by the event.target.value. I recommend you console.log both of the values, above the setState, so you understand.
 
-  const handleChange = (event: ChangeEvent<HTMLInputElement | HTMLTextAreaElement> | FormEvent<FormControl>) => {
+  const handleChange = (
+    event: ChangeEvent<HTMLInputElement | HTMLTextAreaElement> | FormEvent<any>
```
Comment (jeremydthomas) [inline src/profile/EditProfile.tsx:81]: Lets try to type the <any> in FormEvent<any> if possible.

File: src/profile/EditProfile.tsx [hunk 3]
```diff
@@ -97,15 +110,10 @@ const EditProfile = (props: any) => {
     setState((prevState) => ({ ...prevState, isLoading: false }));
   };
 
-  const editName = async (e: FormEvent<HTMLFormElement>) => {
-    e.preventDefault();
-    let body = {
-      fName: state.fName,
-      lName: state.lName,
-    };
+  const editName = async (data: any) => {
```
Comment (jeremydthomas) [inline src/profile/EditProfile.tsx:113]: type out the any in (data: any)  please.

File: src/profile/EditProfile.tsx [hunk 4]
```diff
@@ -97,15 +110,10 @@ const EditProfile = (props: any) => {
...
     try {
       const newName = await RestAPI.put("pareto", `/users/${state.id}`, {
-        body,
+        body: data,
```
Comment (jeremydthomas) [inline src/profile/EditProfile.tsx:116]: good job adding the type here!

**Review by jeremydthomas — COMMENTED** [review]
Overall, this looks pretty awesome! Great job, just fix those few things I put in there.

File: src/profile/EditProfile.tsx (hunk 1 above)
Comment (aldovandus) [reply src/profile/EditProfile.tsx:70]: Done

**Review by jeremydthomas — APPROVED** [review]

**Comment by mikhael28** [comment]
🙏

**Pull request merged**